CHANGELOG.md

```diff
@@ -93,6 +93,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - file-browser-item now supports icons for different file extensions
 - dropzone-widget now has custom dropzone class that extends dropzone functionality to conditionally disallow folders and multiple files from being dropped
 - 'Search' button in navbar to link to search page
+- Consolidate logic for serializing dirty relationships into `osf-serializer`
+  - Override `relationshipTypes` in a serializer to include `fieldName: 'apiType'` pairs of all relationships which may be included when saving updates
```

Comment:
These changes are already in develop (see three lines down), could you make sure this PR is based on the right branch/commit of ember-osf?